cli/smartcontract/generate.go

```diff
@@ -17,7 +18,8 @@ var generatorFlags = []cli.Flag{
 	&cli.StringFlag{
 		Name:    "config",
 		Aliases: []string{"c"},
-		Usage:   "Configuration file to use",
+		Usage: "Configuration bindings file to use (*.yml)." +
+			"Can be generated by 'contract compile' command with --bindings flag",
```

Comment:
Please, follow the format of multi-line usage description of other flags/commands. Preserve the line width and wrap documentation paragraph with backticks. It's needed for better documentation maintaining experience. See as an example the description of `generate-wrapper`.

> Can be generated

Configuration file is generated...

Because there's no other source of this configuration file except this command. No one construct these configs manually.